tests/import_test.go

```diff
@@ -915,6 +915,13 @@ var _ = Describe("Preallocation", func() {
 		tinyCoreRegistryURL = func() string { return fmt.Sprintf(utils.TinyCoreIsoRegistryURL, f.CdiInstallNs) }
 		imageioURL          = func() string { return fmt.Sprintf(utils.ImageioURL, f.CdiInstallNs) }
 		vcenterURL          = func() string { return fmt.Sprintf(utils.VcenterURL, f.CdiInstallNs) }
+		md5PrefixSize       = int64(100000)
+		tinyCoreMD5         = "3710416a680523c7d07538cb1026c60c"
+		tinyCoreTarMD5      = "aec1a39d753b4b7cc81ee02bc625a342"
+		tinyCoreBlockMD5    = "d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
This MD5 sum is used in several places in this file already, since you created the constant, can you use it in the existing tests also?

Reply:
Sure.